doc/examples/dual_arms/dual_arm_panda_moveit_config/config/moveit_controllers.yaml

```diff
@@ -4,34 +4,4 @@ trajectory_execution:
   allowed_goal_duration_margin: 0.5
   allowed_start_tolerance: 0.01
 
-moveit_controller_manager: moveit_simple_controller_manager/MoveItSimpleControllerManager
-
-controller_names:
-  - left_arm_controller
-  - right_arm_controller
-
-left_arm_controller:
-  action_ns: follow_joint_trajectory
-  type: FollowJointTrajectory
-  default: true
-  joints:
-    - left_panda_joint1
-    - left_panda_joint2
-    - left_panda_joint3
-    - left_panda_joint4
-    - left_panda_joint5
-    - left_panda_joint6
-    - left_panda_joint7
-
-right_arm_controller:
-  action_ns: follow_joint_trajectory
-  type: FollowJointTrajectory
-  default: true
-  joints:
-    - right_panda_joint1
-    - right_panda_joint2
-    - right_panda_joint3
-    - right_panda_joint4
-    - right_panda_joint5
-    - right_panda_joint6
-    - right_panda_joint7
+moveit_controller_manager: moveit_ros_control_interface/MoveItMultiControllerManager
```

Comment:
I accidentally used `MoveItMultiControllerManager`, `moveit_ros_control_interface/MoveItControllerManager` should work too